etc/DependencyInstaller.sh

```diff
@@ -215,6 +242,7 @@ while [ "$#" -gt 0 ]; do
             ;;
         -prefix=*)
             OR_INSTALLER_ARGS="${OR_INSTALLER_ARGS} $1"
+            PREFIX=${1#*=}
```

Comment:
What does this do?

Reply:
it assigns everything after `=` to the `PREFIX` variable, which is later used as a Verilator prefix installation